com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,83 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
```

Comment:
one more thing: there is discrepency between the base app and the runtime.

GIven that https://github.com/flathub/org.electronjs.Electron2.BaseApp/issues/29 is open, I think the runtime version should be 20.08 for now.